setup_deb.sh

```diff
@@ -140,6 +140,10 @@ pip3 install --no-deps --upgrade \
    inflect==6.0.4 \
    pydantic==1.10.9
 
+git clone https://github.com/cocodataset/cocoapi.git
```

Comment:
hmm, no way to install from pypi? can you point me to the CI issue? fyi this repo is no longer maintained - there is a fork which is and with which pypi packages are being built: https://github.com/ppwwyyxx/cocoapi

Reply:
You can click on one of the red crosses on the commits above and the issue will be there.